vue-frontend/src/App.vue

```diff
@@ -176,7 +176,7 @@ a {
   font-size: 1rem;
   line-height: 1.25rem;
   letter-spacing: 1px;
-  color: #3d3d3d;
+  color: #3d3d3d !important;
```

Comment:
We should not use `important` if possible

Reply:
i have used .v2-title-3-text  in which you used important in font color hence this overriding my button font color so I need to use !important. Can I remove that !important in  .v2-title-3-text   in  app.vue